src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -125,6 +125,7 @@ private async Task DelayUntilFunctionDispatcherInitializedOrShutdown()
         private async Task<(string name, DataType type, object value)[]> BindInputsAsync(Binder binder)
         {
             var bindingTasks = _inputBindings
+                .AsParallel()
```

Comment:
Wondering if it might be safer to enforce `.WithDegreeOfParallelism` to defined number.

Reply:
That would make sense, I have tested with 3 input bindings and output bindings.
Do you have any preference for the maximum degree of parallelism?